build.sbt

```
@@ -1,12 +1,14 @@
-import StateMachine._
+import StateMachine.*
 import sbtbuildinfo.BuildInfoPlugin.autoImport.{BuildInfoKey, buildInfoKeys}
-import scala.sys.process._
+
+import scala.collection.immutable.Seq
+import scala.sys.process.*
 
 val scroogeVersion = "4.12.0"
 val awsVersion = "1.11.678"
 val awsV2Version = "2.21.17"
 val pandaVersion = "3.0.1"
-val atomMakerVersion = "1.3.4"
+val atomMakerVersion = "2.0.0-PREVIEW.fix-tests-under-java-17.2024-01-18T1039.b4d55b3d"
```

This is a preview release of https://github.com/guardian/atom-maker/pull/94, which should be merged before this PR, so that we can just use a proper full release version:

```suggestion
val atomMakerVersion = "2.0.0"
```